entity-framework/core/extensions/autohistory.md

```diff
@@ -8,12 +8,9 @@ ms.technology: entity-framework-core
 uid: core/extensions/autohistory
 ---
 
-# Microsoft.EntityFrameworkCore.AutoHistory
+# EF Core Tools & Extensions: Microsoft.EntityFrameworkCore.AutoHistory
```

Comment:
Do you prefer just the name of the extension in the title? I can go with that, but if you can think of a a shorter option, let me know. Is "Tools & Extensions: " without "EF Core" any better?